homeassistant/components/switch/wirelesstag.py

```diff
@@ -0,0 +1,129 @@
+"""
+Switch implementation for Wireless Sensor Tags (wirelesstag.net) platform.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/switch.wirelesstag/
+"""
+import logging
+
+import voluptuous as vol
+
+
+from homeassistant.components.wirelesstag import (
+    DEFAULT_ENTITY_NAMESPACE, DOMAIN as WIRELESSTAG_DOMAIN,
+    WIRELESSTAG_TYPE_13BIT, WIRELESSTAG_TYPE_WATER,
+    WIRELESSTAG_TYPE_ALSPRO,
+    WirelessTagBaseSensor)
+from homeassistant.components.switch import PLATFORM_SCHEMA, SwitchDevice
+from homeassistant.const import (
+    CONF_ENTITY_NAMESPACE, CONF_MONITORED_CONDITIONS, STATE_OFF, STATE_ON)
+import homeassistant.helpers.config_validation as cv
+
+DEPENDENCIES = ['wirelesstag']
+
+_LOGGER = logging.getLogger(__name__)
+
+ARM_TEMPERATURE = 'temperature'
+ARM_HUMIDITY = 'humidity'
+ARM_MOTION = 'motion'
+ARM_LIGHT = 'light'
+ARM_MOISTURE = 'moisture'
+
+# Switch types: Name, tag sensor type
+SWITCH_TYPES = {
+    ARM_TEMPERATURE: ['Arm Temperature', 'temperature'],
+    ARM_HUMIDITY: ['Arm Humidity', 'humidity'],
+    ARM_MOTION: ['Arm Motion', 'motion'],
+    ARM_LIGHT: ['Arm Light', 'light'],
+    ARM_MOISTURE: ['Arm Moisture', 'moisture']
+}
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Optional(CONF_ENTITY_NAMESPACE, default=DEFAULT_ENTITY_NAMESPACE):
```

Comment:
This is already part of PLATFORM_SCHEMA.

Reply:
@balloob got it. removing...